example/src/pages/Icons.tsx

```diff
@@ -0,0 +1,20 @@
+
+import React from 'react'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faUpload } from '@fortawesome/free-solid-svg-icons'
+
+const IconsPage = (): React.ReactElement => (
+  <section>
+    <h1>Icons</h1>
+
+    <p className="usa-intro">
+      We recommend using icons from <a href="https://www.google.com/search?client=safari&rls=en&q=react-fontawesome&ie=UTF-8&oe=UTF-8">react-fontawesome</a> for projects that need icons.
```

Comment:
did you mean to link to the google search results?

Reply:
LOL nope!